src/design-tokens/breakpoints.js

```diff
@@ -0,0 +1,4 @@
+export const XL = 1720
+export const L = 1024
+export const M = 768
+export const S = 420
```

Comment:
If you haven't seen the way we are doing this on the styleguide I think it might be something worth bringing into this library